pom.xml

```diff
@@ -344,14 +256,11 @@
         <commons-collections.wso2.osgi.version.range>[3.2.0,4.0.0)</commons-collections.wso2.osgi.version.range>
         <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
-        <io.asgardio.java.saml.sdk.version>0.1.0</io.asgardio.java.saml.sdk.version>
+        <io.asgardio.java.saml.sdk.version>0.1.1-SNAPSHOT</io.asgardio.java.saml.sdk.version>
```

Comment:
Don't have snapshot versions of dependencies.

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993